modules/home/assets/js/app.js

```diff
@@ -8,26 +8,25 @@ const App = ( props ) => {
 	return (
 		<DirectionProvider rtl={ props.isRTL }>
 			<LocalizationProvider>
-				<ThemeProvider colorScheme={ props.colorScheme }>
-					<HomeScreen />
+				<ThemeProvider colorScheme={ 'light' }>
+					<HomeScreen homeScreenData={ props.homeScreenData } />
 				</ThemeProvider>
 			</LocalizationProvider>
 		</DirectionProvider>
 	);
 };
 
 const isRTL = elementorCommon.config.isRTL,
-	rootElement = document.querySelector( '#e-home-screen' ),
-	colorScheme = rootElement?.dataset?.ui_theme || 'auto';
+	rootElement = document.querySelector( '#e-home-screen' );
 
 App.propTypes = {
-	colorScheme: PropTypes.oneOf( [ 'auto', 'light', 'dark' ] ),
 	isRTL: PropTypes.bool,
+	homeScreenData: PropTypes.object,
 };
 
 ReactUtils.render( (
 	<App
-		colorScheme={ colorScheme }
```

Comment:
We realized that WordPress doesn't support the dark mode. Therefore we shouldn't activate it on this page, because it will look odd.